packages/matchbox/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Columns } from '../Columns';
+
+import Section from './Section';
+import SectionTitle from './SectionTitle';
+
+const breakpoints = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+const Layout = React.forwardRef(function Layout(props, ref) {
+  const { children, collapseBelow } = props;
+
+  return (
+    <Columns collapseBelow={collapseBelow} space="500" mb="36px" ref={ref}>
```

Comment:
Also I would make `space` responsive as well based on our discussion today – i would go down to `400` on smaller screens